packages/deployment/app/cloudformation.template.yml

```diff
@@ -984,8 +984,8 @@ Resources:
     {{/if}}
       LaunchConfigurationName:
         Ref: CumulusContainerInstanceLaunch
-      MinSize: '0'
-      MaxSize: {{ ecs.maxInstances }}
+      MinSize: {{#if ecs.minInstances }}{{ ecs.minInstances }}{{ else }}'0'{{/if}}
+      MaxSize: {{#ifLessThan ecs.maxInstances 11 }}{{ ecs.maxInstances }}{{ else }}10{{/ifLessThan}}
```

Comment:
So we're capping it at 10?

Reply:
This has been reverted based on @bilts comment we just need maxInstances to be configurable